crates/compiler/load_internal/src/file.rs

```diff
@@ -2986,8 +2986,10 @@ fn update<'a>(
 fn register_package_shorthands<'a>(
     shorthands: &mut MutMap<&'a str, ShorthandPath>,
     package_entries: &MutMap<&'a str, header::PackageName<'a>>,
+    #[allow(unused_variables)] // for wasm
     module_path: &Path,
     src_dir: &Path,
+    #[allow(unused_variables)] // for wasm
     cache_dir: &Path,
```

Comment:
`allow unused` seems like the best solution here, I tried `cfg non wasm` first but that would have required me to change every call as well. That adds too much noise to the code in my opinion.